frontend/app/src/components/home/ChatMessageMenu.svelte

```diff
@@ -260,13 +290,66 @@
             }
         });
     }
+
+    // Note: Manually selected reactions do not increment their fav counter!
+    function toggleReaction(reaction: string) {
+        if (canReact) {
+            const kind = client.containsReaction(createdUser.userId, reaction, msg.reactions)
+                ? "remove"
+                : "add";
+
+            client
+                .selectReaction(
+                    chatId,
+                    createdUser.userId,
+                    threadRootMessageIndex,
+                    msg.messageId,
+                    reaction,
+                    createdUser.username,
+                    createdUser.displayName,
+                    kind,
+                )
+                .then((success) => {
+                    if (success && kind === "add") {
+                        client.trackEvent("reacted_to_message");
+                    }
+                });
+        }
+    }
 </script>
 
-<div class="menu" class:rtl={$rtlStore}>
+<div class="menu" class:inert={inert} class:rtl={$rtlStore}>
+    {#await loadedReactions }
+        <div></div>
```

Comment:
Given that we don't have any meaningful placeholder while waiting here we should just use the shorter syntax: 

```
{#await loadedReactions then [fst, snd, thrd]}
```